Python/Product/VSInterpreters/Interpreter/CondaEnvironmentManager.cs

```diff
@@ -291,8 +309,15 @@ sealed class CondaEnvironmentManager : ICondaEnvironmentManager, IDisposable {
             }
         }
 
+        private async Task EnsureActivatedAsync() {
+            if (_activatedEnvironmentVariables == null) {
+                var env = await CondaUtils.GetActivatedEnvironmentVariablesAsync(CondaPath);
```

Comment:
If "activate.bat" does not exist in the Conda environment, it will return an empty KeyValuePair [Line 82]. Should we be error checking here? Same for other cases relating to "GetActivatedEnvironmentVariables", "ParseEnvironmentVariables", etc.

Reply:
I made it work this way because:
1. Returning empty environment variables in the case of failing activation is "safer" and falls back to old behavior (it will use VS' process environment variables). I prefer safer solution right now before release (no unanticipated crash due to missing error handling)
2. Some scenarios will work without activation, so forcing a failure could make things worse ie. block the user when things would work just fine.
3. Failing to activate will lead to a known class of errors "failed to load module" that are easy for us to identify.
4. We're past the localization deadline so I'd prefer not introducing error messages at this time if possible/reasonable.

Reply:
3. If the activation fails, will this known class of errors display an error stating that activate environment failed? My concern is that we would confuse the user if we don't display some kind of error message whenever we fail to create the environment.

Reply:
The user gets an error message right now because we don't activate. With this change we will continue to display the same error message if activation fails. So this won't be more confusing than it is now.